src/ts/deploy.ts

```diff
@@ -278,7 +280,17 @@ export async function prepareVirtualDeploymentFromSafe(
 export async function getDeployArgsFromRealToken(
   realToken: Contract,
 ): Promise<Omit<RealTokenDeployParams, "totalSupply">> {
+  const filterMintingTransfers = realToken.filters.Transfer(
+    "0x0000000000000000000000000000000000000000",
+    null,
+    null,
+  );
```

Comment:
Nice! Only a small number of events should ever match this filter, so this should always be working.